src/test/tcp/test_tcp.c

```diff
@@ -20,12 +20,19 @@
 #include <sys/uio.h>
 #include <time.h>
 #include <unistd.h>
+#include <glib.h>
+#include <sys/msg.h>
+#include <sys/types.h>
+#include "test/test_glib_helpers.h"
```

Comment:
Please put a line-break between the system headers and our internal includes. This helps keep them in the right order when using e.g. `clang-format`.